Shady nagy/add date created admin

ShadyNagy opened the pull request:
- `Need` DB changes identity only
Fix #144

ardalis commented:
Note: migrations don't work so I added the column by hand. I think the issue may be that the AspNetUsers type isn't part of my solution, but not sure.

ShadyNagy commented:
I think At some point, we need to make PR for migration

ShadyNagy commented:
> Note: migrations don't work so I added the column by hand. I think the issue may be that the AspNetUsers type isn't part of my solution, but not sure.

Fixed on PR #842
